.gitignore

```diff
@@ -12,3 +12,10 @@
 
 # Project-local glide cache, RE: https://github.com/Masterminds/glide/issues/736
 .glide/
+
+vendor
+
+*.results
+
+glide
+easyjson
```

Comment:
Why is easyjson in the ignore? Just curious, I might not be clicking why this is needed :)

Reply:
it is not recommended to commit the binary, easyjson is a command for generate easyjson go file.

Reply:
Oh gotcha! Okay I think if we put the editor config in another PR I'm happy with contributing this.

Reply:
Sorry for the slow response on this. I _really_ appreciate your efforts :) Thank you SO much!